docs/mddocs/Quickstart/ollama_portable_zip_quickstart.md

```diff
@@ -224,7 +226,7 @@ The currently Ollama Portable Zip is based on Ollama v0.5.4; in addition, the fo
 
 ### Signature Verification
 
-For portable zip/tgz version 2.2.0, you could verify its signature with the following command:
```

Comment:
No need to change for signature verification. As only 2.2.0 has signature for now